src/artm/version.h

```diff
@@ -0,0 +1,9 @@
+// Copyright 2016, Additive Regularization of Topic Models.
```

Comment:
Please exclude this file from your check-in. It is auto-generated as part of the build system. Feel free to add it into ``.gitignore``.